### ISSUE TITLE
[Tip] Invoke-WebRequest in AU update script

### ISSUE THREAD

**Issue by AdmiringWorm** [chocolatey/chocolatey-packages-template#10]
Just came across this when I was creating some packages using AU.

If you are using `Invoke-WebRequest` in the AU update script,
and one of the following happens:

* When invoking the `update.ps1` script manually it seems to hang and possibly throws an operation timed out.
* When launching the `update_all.ps1` script certain packages are never updated, they just show an empty version.
* The following windows security warning dialogbox shows when starting from the PowerShell ISE:  
![image](https://cloud.githubusercontent.com/assets/1474648/18225759/3b261174-71fb-11e6-9325-518eba3bf11b.png)

If any of these happen ty changing `Invoke-WebRequest` to `Invoke-WebRequest -UseBasicParsing`

**Comment by ferventcoder** [comment]
Great tip!

**Comment by ferventcoder** [comment]
Definitely important for folks that don't want to open and configure Windows Internet Options as well

**Comment by AdmiringWorm** [comment]
So true, Personally I didn't even think of the posibility of configuring Windows Internet Options to get rid of the warning.

Also re-read the documentation of the cmdlet and found one piece of information I missed when reading it for the first time.

> This parameter is required when Internet Explorer is not installed on the computers, such as on a Server Core installation of a Windows Server operating system.

**Comment by ferventcoder** [comment]
@majkinetor I think you may have already handled this one?

**Comment by majkinetor** [comment]
Yes, seems like a `iwr` bug reported by several people. Details:

[Invoke-WebRequest can block the updater](https://github.com/majkinetor/au/wiki/Troubleshooting)

**Comment by ferventcoder** [comment]
Is it bad to add the `-UseBasicParsing` to the default?

**Comment by majkinetor** [comment]
I don't know. 

Some sites work with it, some without it. Its even machine dependent ?! On my local machine update runs, then on AV it doesn't. When I set it to run on AV, it doesn't run on  my local machine. 

Its not that relevant since you don't have to use it with AU, its just damn convenient.

**Comment by ferventcoder** [comment]
Basically if you have not opened Internet Explorer or it's options, you have to set `-UseBasicParsing` or IWR just doesn't work.

**Comment by majkinetor** [comment]
Thats strange and i doubt it. How my packages function then on AV? Also this behavior would make iwr not working on core.

**Comment by majkinetor** [comment]
I checked online docs and it never mentions this as far as I can see. Also, cookies are not saved by default, perhaps this behavior is only true when u use `SessionVariable` ?
